tests/Services/Validators/EmailValidatorTest.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Tests\Services\Validators;
+
+use App\Services\Validators\EmailValidator;
+use Tests\TestCase;
+
+class EmailValidatorTest extends TestCase
+{
+    public function testValidateEmailSuccess()
+    {
+        $result = EmailValidator::validateEmail('Example@example.com');
+        $this->assertEquals($result, 'Example@example.com');
+    }
+
+    public function testValidateEmailFailure()
+    {
+        $result = EmailValidator::validateEmail('Example@examplecom');
+        $this->assertEquals($result, false);
+    }
```

Comment:
Should we do a malformed test here as well?

Reply:
Extra test added testing array insertion into validator